sr_ur_launch/scripts/teach_mode_node

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+
+from controller_manager_msgs.srv import (ListControllers, LoadController,
+                                         SwitchController, SwitchControllerRequest)
+import rospy
+from sr_robot_msgs.msg import ControlType
+from sr_robot_msgs.srv import (ChangeControlType, RobotTeachMode,
+                               RobotTeachModeRequest, RobotTeachModeResponse)
+from sr_ur_msgs.srv import SetTeachMode
+
+
+class ControllerHelper(object):
+
+    def __init__(self, robot_ids, robot_joint_prefixes):
+        self.time_to_reload_params = 6.0
+        self.robot_ids = robot_ids
+        self.robot_joint_prefixes = robot_joint_prefixes
+
+        # This section will only be used for hands
+        self.joints = ["ffj0", "ffj3", "ffj4",
```

Comment:
We will have to adapt this to be compatible with hand H. We need to add this to our Jira backlog.